### PR TITLE
Add cpp-sort 1.5.1

### PR DESCRIPTION
**Pull request by Morwenn**
Specify library name and version:  **cpp-sort/1.5.1**

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/wiki#how-to-submit-a-pull-request) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Comment by Morwenn** [comment]
I asked for Early Access rights in the appropriate section, I'm uploading this recipe here in the meantime to make sure I don't accidentally lose it or forget about it.

**Comment by danimtb** [comment]
Don't worry @Morwenn. We are currently experiencing some issues in the build service rebuilding mac packages. It is totally fine to open the PR. We will add new users to the EAP as soon as we have the packages updated. Thanks!

**Comment by Morwenn** [comment]
I realize that the `exports` in my recipe is most likely unneeded (albeit not dangerous) but the continuous integration is running. Is it safe to change my merge request while CI is running?

**Comment by danimtb** [comment]
yes, don't worry. If you push new commits, the previous jobs that are already running will be canceled automatically

**Comment by Morwenn** [comment]
Oh, looks like I forgot that things like `cxx_std_14` weren't propagated through Conan `cmake` generator, and it just happened to work at home thanks to my compiler defaulting to `-std=gnu++14`. I have to make this explicit in `test_package/CMakeLists.txt` somehow.

Also it looks like I uploaded the package with the wrong user information (my mistake for doing that from work). The guy mentioned isn't me and probably wouldn't like his name here, plus he won't sign the CLA - is it possible to get rid of his info? I'm sorry for the mess -_-

**Comment by danimtb** [comment]
@Morwenn I guess the best option is to remove that commit keeping the changes and force push the changes. CLA should update the info accordingly

**Comment by Morwenn** [comment]
@danimtb Ok, thanks for the information, I will update the recipe later tonight when I'm back home in order to avoid making further mistakes.

**Comment by Morwenn** [comment]
Ok, I forced pushed and apparently fixed everything. It should be all good now :)

File: recipes/cpp-sort/all/conanfile.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+from conans import CMake, ConanFile, tools
+
+
+class CppSortConan(ConanFile):
+    name = "cpp-sort"
+    description = "Additional sorting algorithms & related tools"
+    topics = "conan", "cpp-sort", "sorting", "algorithms"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/Morwenn/cpp-sort"
+    author = "Morwenn <morwenn29@hotmail.fr>"
```
Comment (SSE4) [inline recipes/cpp-sort/all/conanfile.py]:
```suggestion

```

File: recipes/cpp-sort/all/test_package/conanfile.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python
```
Comment (SSE4) [inline recipes/cpp-sort/all/test_package/conanfile.py]:
```suggestion
```

File: recipes/cpp-sort/all/test_package/conanfile.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+# -*- coding: utf-8 -*-
```
Comment (SSE4) [inline recipes/cpp-sort/all/test_package/conanfile.py]:
```suggestion
```

**Comment by Morwenn** [comment]
Oops, I thought that the hooks would take care of those too. Removed.

**Pull request merged**